taf/developer_tool.py

```diff
@@ -91,50 +85,31 @@ def add_signing_key(
     if pub_key_pem is None:
         pub_key_pem = new_public_key_cmd_prompt(scheme)["keyval"]["public"]
 
-    if taf_repo.is_valid_metadata_key(role, pub_key_pem):
-        print(f"Key already registered as signing key of role {role}")
-        return
+    for role in roles:
 
-    taf_repo.add_metadata_key(role, pub_key_pem, scheme)
+        if taf_repo.is_valid_metadata_key(role, pub_key_pem):
+            print(f"Key already registered as signing key of role {role}")
+            return
```

Comment:
Should we continue registering a key for other roles, instead of stopping the iteration?

Reply:
Yes, will fix